lua/terrortown/lang/en.lua

```diff
@@ -1983,6 +1983,43 @@ mode 1: This mode increases the importance of public policing roles by limiting
 
 mode 2: This mode is yet a bit more strict than mode 1. In this mode the search ability is removed as well from normal players. This means that reporting a dead body to a public policing player is now the only way to get any information from dead bodies.]]
 
+-- 2023-10-19
+L.label_grenade_trajectory_ui = "Enable grenade trajectory indicator"
+L.label_grenade_fuse_meter_ui = "Enable grenade detonation fuse meter"
+
+L.label_ttt_firegrenade_proj_explosion_radius = "Explosion Radius"
+L.help_ttt_firegrenade_proj_explosion_radius = "The range in which players will be damaged instantly upon the projectile exploding."
+L.label_ttt_firegrenade_proj_explosion_damage = "Explosion Damage"
+L.help_ttt_firegrenade_proj_explosion_damage = "The amount of damage to apply to players within range instantly upon the projectile exploding."
+L.label_ttt_firegrenade_proj_fire_num = "Fires Quantity"
+L.help_ttt_firegrenade_proj_fire_num = "The number of fires that should be spawned upon the projectile exploding."
+L.label_ttt_firegrenade_proj_fire_lifetime = "Fires Lifetime"
+L.help_ttt_firegrenade_proj_fire_lifetime = "How long fires should live upon the projectile exploding."
+
+L.label_ttt_smokegrenade_proj_smoke_life_time = "Lifetime"
+L.help_ttt_smokegrenade_proj_smoke_life_time = "The duration that a smoke will last for, not including the fade time."
+
+L.menu_thirdparty_title = "Third-Party"
+L.menu_thirdparty_description = "Features for third-party addons which are supported by TTT2."
+
+L.submenu_thirdparty_general_title = "General"
+L.header_thirdparty_general = "General"
+
+L.label_thirdparty_enable_workshop = "Enable Workshop"
+L.help_thirdparty_enable_workshop = "Allow TTT2 to AddWorkshop supported Third-Party addons, providing the ideal gameplay experience."
```

Comment:
This should be rewritten because we can't `AddWorkshop` on the server addons anyway. Is this string still used anywhere? Because the convar was removed